packages/result/src/lib/Parsers.ts

```diff
@@ -0,0 +1,34 @@
+import { err, ok, Result } from './Result';
+
+type Awaitable<T> = PromiseLike<T> | T;
```

Comment:
Also put this type below

So the structure is

- from
- fromAsync
- isFunction
- Awaitable